src/client/components/Tooltip/Tooltip.tsx

```diff
@@ -84,11 +84,11 @@ const TooltipText = styled.p`
 `
 
 export const Tooltip: React.FC<TooltipProps> = ({ body }) => {
-  const ref = React.useRef<HTMLDivElement>(null)
-  const [isVisible, setVisible] = React.useState(false)
+  const ref = useRef<HTMLDivElement>(null)
+  const [isVisible, setVisible] = useState(false)
 
-  React.useEffect(() => {
-    if (isVisible === false) return
+  useEffect(() => {
+    if (!isVisible) return
```

Comment:
nit: is this really more readable than what we had before? ;)

Reply:
🤔 I think so. I mean, it's almost like reading plain english: "if not visible". But I can revert it, no problem at all.

Reply:
aha interesting 😅 I read it as "if not _is_ visible" which makes me confused -- while the other one is "if is visible is false". But don't change it unless other people feel the same as me hehe

Reply:
I think when we use the `is` prefix for booleans, I have no trouble reading it like `!isVisible`. So I approve 🤗